lib/instrumentation/grpc-js/grpc.js

```diff
@@ -217,6 +213,11 @@ module.exports = function instrument(shim) {
   genericShim.wrap(callStream.Http2CallStream.prototype, 'start', wrapStart)
 
   const webFrameworkShim = shim.makeSpecializedShim(shim.WEB_FRAMEWORK, 'server')
+  const grpcVersion = shim.require('./package.json').version
+  if (semver.lt(grpcVersion, '1.4.0')) {
+    shim.logger.debug('gRPC server-side instrumentation only supported on grpc-js >=1.4.0')
```

Comment:
you need to return otherwise it's just going to log this and resume instrumenting the server

Reply:
Right right